storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -29,6 +29,44 @@ In future {product-title} versions, volumes provisioned using existing in-tree p
 After full migration, in-tree plug-ins will eventually be removed in future versions of {product-title}.
 ====
 
+=== Storage policy
+
+The vSphere CSI Operator Driver storage class now uses vSphere's storage policy. {product-title} automatically creates a storage policy that targets datastore configured in cloud configuration:
+[output, yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: thin-csi
+provisioner: csi.vsphere.vmware.com
+parameters:
+  StoragePolicyName: "${STORAGE_POLICY_NAME}"
+volumeBindingMode: WaitForFirstConsumer
+# Volume expansion is disabled here because volume expansion in the vSphere
+# CSI driver is still in beta and has several limitations. (see
+# https://vsphere-csi-driver.sigs.k8s.io/supported_features_matrix.html).
+# It is not supported for all datastores (vSAN only right now) and there is
+# not yet an e2e test for it.
+allowVolumeExpansion: false
+reclaimPolicy: Delete
+----
+
+=== Version requirements
+
+The vSphere CSI Operator Driver requires certain minimum versions for products that make up the virtual environment as shown in the following table.
+
+.Version requirements for vSphere CSI Operator Driver
+[cols=", ^v, width="100%", options="header"]
+|===
+|Virtual environment product |Required version
+
+|VM hardware version | 15 or later
+|vSphere ESXi hosts | 6.7u3 or later
+|vCenter host   | 6.7u3 or later
+|===
```

Comment:
These version requirements are also becoming a general requirement for OCP in 4.9. Customers will get alerts regardless of the fact that - whether they are using CSI driver or not. So, we should perhaps also document the alerts mentioned in https://github.com/openshift/cluster-storage-operator/pull/209/files in https://docs.openshift.com/container-platform/4.8/installing/installing_vsphere/using-vsphere-problem-detector-operator.html and what those alerts mean.

Reply:
also cc @codyhoag